src/runner/task-status.js

```diff
@@ -0,0 +1,5 @@
+export default {
+    done:     'task-done',
```

Comment:
If we call it `status` then where should be a value for all phases of the browser job. E.g. `running` and `pending` are missing. However, if we'll rename it to "browser job result" these values will be legit.